test/stringify.js

```diff
@@ -37,7 +37,7 @@ test('should not encode undefined values', t => {
 });
 
 test('should encode null values as just a key', t => {
-	t.deepEqual(fn.stringify({xyz: null, abc: null, foo: 'baz'}), 'abc&foo=baz&xyz');
+	t.deepEqual(fn.stringify({'x y z': null, 'abc': null, 'foo': 'baz'}), 'abc&foo=baz&x%20y%20z');
```

Comment:
All I did was augment an existing test, btw. If you want to use `t.is` do you want me to change all the other tests too while I'm at it?